src/containers/audio-rewards-page/components/modals/TransferAudioMobileDrawer.tsx

```diff
@@ -0,0 +1,26 @@
+import Drawer from 'components/drawer/Drawer'
+import React from 'react'
+import { useModalState } from 'store/application/ui/modals/hooks'
+import { ReactComponent as IconGold } from 'assets/img/IconGoldBadge.svg'
+import styles from './TransferAudioMobileDrawer.module.css'
+
+const messages = {
+  title: 'Transfer $AUDIO',
+  subtitle: 'To transfer AUDIO please visit audio.co from a desktop browser'
```

Comment:
audius.co

Reply:
also maybe copy shold be to transfer $AUDIO?

Reply:
heh